src/test/java/pdfParsers/PdfToJsonConverterTest.java

```diff
@@ -0,0 +1,121 @@
+package pdfParsers;
+
+import loaders.LocalPdfReader;
+
+import java.io.InputStream;
+import java.io.IOException;
+
+import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.JsonParser;
+import org.codehaus.jackson.map.MappingJsonFactory;
+import org.codehaus.jackson.node.ArrayNode;
+import org.junit.Test;
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeDiagnosingMatcher;
+
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class PdfToJsonConverterTest {
+    
+    @Test
+    public void checkboxPdfHasExpectedFields() throws Exception {
+        assertThat(getFieldsFromPdf("checkbox.pdf"),
+            contains(
+                allOf(
+                    hasJsonProperty("name", equalTo("Check Box2")),
+                    hasJsonProperty("value", equalTo("")),
+                    hasJsonProperty("type", equalTo("checkbox")),
+                    hasJsonProperty("tabIndex", equalTo(0)),
+                    hasJsonProperty("required", equalTo(false)),
+                    hasJsonProperty("altText", equalTo(""))
```

Comment:
Ben asked if I could read and return altText.  This makes it easier for someone to rebuild the form in HTML or mobile app and present it to the end user.  The end user fills out the form and submits it.